host/testcluster.go

```diff
@@ -56,6 +56,7 @@ type (
 		NumHistoryShards      int
```

Comment:
Can you check if you can move these two into the history config?

Reply:
Ok, updated